test/test_device_thermal_generation_constructors.jl

```diff
@@ -826,7 +826,7 @@ end
     min_limit = PSY.get_active_power_limits(comp).min
     @test isapprox(
         PSI.no_load_cost(
-            VariableCost(5.0),
+            PSY.LinearFunctionData(5.0),
```

Comment:
@GabrielKS What is the correct change for line 838?

Reply:
Line 838 should be `PSY.QuadraticFunctionData(3.0, 5.0, 0.0)`.

You also want to change https://github.com/NREL-Sienna/PowerSimulations.jl/blob/4f6eb9717e4594d551329ceb685b1172d26075a8/src/devices_models/devices/thermal_generation.jl#L128 to `return evaluated * PSY.get_system_base_power(d)`, this is another symptom of the merge

Reply:
Thanks. Fixed now.